main.py

```diff
@@ -200,6 +204,85 @@ def randomchat(bot, update):
         bot.sendMessage(update.message.chat_id, text=reply)
 
 
+def estanoche(bot, update):
+    # Initalize coordinates
+    lan = None
+    lat = None
+
+    # TODO pass location as argument an resolve coordinates
+
+    if not lan or not lat:
+        # Yeah, Robledo de Chavela coordinates by default
+        lat = 40.498333
+        lon = -4.238889
+
+    # Building URL to query the service
+    url_service = 'http://202.127.24.18/bin/astro.php?lon='
+    url_params = '&ac=0&unit=metric&output=json&tzshift=0'
+    url_gen = url_service + str(lon) + '&lat=' + str(lat) + url_params
+
+    # Query service
+    timer7 = requests.get(url_gen)
+    if timer7.status_code > 299:
+        bot.sendMessage(update.message.chat_id, text='El servicio de informacion astronomica no esta disponible.')
+        return
+
+    json_timer7 = timer7.json()
+
+    timer7_data = json_timer7["dataseries"][1]
```

Comment:
Checking the keys for the JSON data (maybe there's an error and, instead of `dataseries` returns `errorcode` and this will throw an Exception) an there's also a list, you should check the length so it has enough elements to reach the one you read.